### DISCUSSION TITLE
Databricks Unity Catalog Compatibility

### DISCUSSION THREAD

**Discussion by james-d-cole** [nightscape/spark-excel#834] [Ideas]
Hi,

I had been successfully using this package to read .xlsx files into Databricks. However, we have recently implemented Unity Catalog (UC) and this package is not compatible with the UC permissions model.

The package expects a legacy ACL permission on the external location and gives the following error when trying to read to a spark data frame...

_[INSUFFICIENT_PERMISSIONS] Insufficient privileges: User does not have permission SELECT on any file._

This permission would bypass all UC permissions which we can not do due to data access requirements.

Are there any plans to make this package compatible with the Databricks Unity Catalog permissions model?

**Comment by nightscape** [comment]
I'm not actively working on the project, but we do accept contributions!
Is anybody on your team familiar with Scala?
